The labour camp shuttle paroles the user when they complete their points. The station return message specifies which prisoner returned.

Pickle-Coding opened the pull request:
## About The Pull Request
The labour camp shuttle paroles the user when they complete their points. The station return message specifies which prisoner returned.
## Why It's Good For The Game
People shouldn't be looking like fugitives after completing their work.
## Changelog
:cl:
qol: The labor camp shuttle properly sets people to parole after they complete their work.
qol: The labor camp shuttle specifies which person returned to the station.
/:cl:

Pickle-Coding committed:
Might be better this way.

Makes the labor camp stacker discharge the user when they complete their points. Also informs security of who completed it.

Iamgoofball commented:
Set to Parole instead; Discharge is for cleared of charges

MrMelbert commented on code/modules/mining/laborcamp/laborstacker.dm:
```diff
@@ -106,7 +106,13 @@
 					if(!(obj_flags & EMAGGED))
 						security_radio.set_frequency(FREQ_SECURITY)
-						security_radio.talk_into(src, "A prisoner has returned to the station. Minerals and Prisoner ID card ready for retrieval.", FREQ_SECURITY)
+						if(!isnull(GLOB.manifest.general))
+							for(var/datum/record/crew/record as anything in GLOB.manifest.general)
+								if(record.name == user_mob.real_name)
+									record.wanted_status = WANTED_DISCHARGED
+									break
```
Use `/proc/find_record`

Pickle-Coding committed:
I am dumb.

Pickle-Coding committed:
It's better this way.

Uses the find_record() proc for finding the user's records instead of doing it manually.

MrMelbert commented on code/modules/mining/laborcamp/laborstacker.dm:
```diff
@@ -106,7 +106,10 @@
 						security_radio.set_frequency(FREQ_SECURITY)
-						security_radio.talk_into(src, "A prisoner has returned to the station. Minerals and Prisoner ID card ready for retrieval.", FREQ_SECURITY)
+						var/datum/record/crew/target = find_record(user_mob.real_name)
+						target.wanted_status = WANTED_PAROLE
```
```suggestion
						target?.wanted_status = WANTED_PAROLE
```

Pickle-Coding committed:
Update code/modules/mining/laborcamp/laborstacker.dm

Co-authored-by: MrMelbert <51863163+MrMelbert@users.noreply.github.com>
